scripts/build.py

```diff
@@ -648,7 +724,19 @@ def generate_build_directory():
     save_hashes_as_json(HASHES_JSON, hashes)
 
     # Minify all template files copy them into build/templates/head.
-    build_files(TEMPLATES_DEV_DIR_CORE, TEMPLATES_STAGING_DIR, hashes)
+    if not os.path.isdir(TEMPLATES_STAGING_DIR):
+        print 'Creating new %s folder' % TEMPLATES_STAGING_DIR
+        build_files(TEMPLATES_DEV_DIR_CORE, TEMPLATES_STAGING_DIR, hashes)
+    else:
+        # Always rebuild HTML files.
```

Comment:
And here: "If there is an existing staging dir, rebuild all HTML files. For other files, rebuild only the ones that have changed since the last build took place." (or something similar)

Reply:
Done!